lib/generators/interface_generator.rb

```diff
@@ -35,7 +35,16 @@ def underscore(word)
   end
 
   def methods
-    args[2..-1].map { |m| ":#{m}" }.join(', ')
+    methods = args[2..-1].map do |m|
+                if m.scan(/\d+/).empty?
+                  ":#{m}"
+                else
+                  m_copy = m.dup
+                  m_copy[":"] = ": " # Create and modify a copy because 'm' is frozen in runtime.
+                  m_copy
```

Comment:
Would it read better to just write:
```ruby
else
  m.gsub(':', ': ')
end
```

which should return a copy.

Also, you could remove `methods = ` and just do `end.join(', ')` at the end of the else